api/tests/routes/api/v1/preference_test.py

```diff
@@ -32,6 +32,8 @@ def test_preferences_api_user_exists(app, database, fake_user):
                 {
                     'active': True,
                     'date': '2017-01-20T23:00:00+00:00',
+                    # TODO(areilly) Where is this coming from?
```

Comment:
Since `active` is true for the subscription we should be adding in the auto_renew value right?